tests/functional/externalpages/analytics/test_google_analytics.py

```diff
@@ -0,0 +1,49 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.externalpages.analytics.index import AnalyticsTestPage
+
+
+def assert_ga_loaded(selenium):
+    # if the datalayer list doesnt contain the gtm.load event then GA has not been loaded or it has been blocked
+    data_layer = selenium.execute_script("return window.dataLayer")
+    assert any(
+        "event" in layer and layer["event"] == "gtm.load" for layer in data_layer
+    ), "These tests require Google Analytics to be configured. \
+       Please ensure you have set a GTM_CONTAINER_ID environment variable in your .env and you are not blocking GA in your browser."
+
```

Comment:
Nano-nit: Multi-line strings should ideally be split by putting them in parentheses, rathe than using a backslash

eg
```
), (
    "These tests require Google Analytics to be configured. "
    "Please ensure you have set a GTM_CONTAINER_ID environment "
    "variable in your .env and you are not blocking GA in your browser."
)
```

Reply:
ah! good to know, thanks!